test/e2e_node/local_storage_isolation_eviction_test.go

```diff
@@ -96,7 +96,7 @@ var _ = framework.KubeDescribe("LocalStorageCapacityIsolationEviction [Slow] [Se
 							Command: []string{
 								"sh",
 								"-c",
-								"dd if=/dev/urandom of=target-file of=/cache/target-file bs=50000 count=1; while true; do sleep 5; done",
+								"sleep 30; dd if=/dev/urandom of=target-file of=/cache/target-file bs=50000 count=1; while true; do sleep 5; done",
```

Comment:
How long does it usually take? Can we add a shorter sleep? I would prefer the sleep to be less than 5s if possible, ideally 1-2s.